packages/core/src/components/AutocompleteField/AutocompleteField.jsx

```diff
@@ -0,0 +1,198 @@
+                        key={item.id}
+                        role="option"
+                        {...getItemProps({ item })}
+                      >
+                        {itemToString(item)}
+                      </li>
+                    ))}
+                </ul>
+              </div>
+            ) : null}
+
+            <Button
+              aria-label={clearAriaLabel}
+              className="ds-u-float--right ds-u-padding-right--0 ds-c-autocomplete__button"
+              href="javascript:void(0);"
+              onClick={clearSelection}
+              size="small"
+              variation="transparent"
+            >
+              {clearInputText}
+            </Button>
+          </div>
+        )}
+      />
+    );
+  }
+}
+
+AutocompleteField.defaultProps = {
+  clearAriaLabel: 'Clear typeahead and search again',
+  clearInputText: 'Search again',
+  constrainedListText: 'Select from the options below:'
+};
+
+AutocompleteField.propTypes = {
+  children: PropTypes.node,
+  /**
+   * Screenreader-specific label for the Clear input link. Intended to provide a longer, more descriptive explanation of the link's behavior.
+   */
+  clearAriaLabel: PropTypes.string,
+  /**
+   * Clear link text that will appear on the page as part of the rendered component
+   */
+  clearInputText: PropTypes.string,
+  /**
+   * Adds a conditional header to `<div.ds-c-autocomplete__list>`
+   */
+  constrainedList: PropTypes.bool,
+  /**
+   * Text string for the `<h5>` header added to the`<div.ds-c-autocomplete__list>` when prop `constrainedList`is passed to `<Autocomplete>`
+   */
+  constrainedListText: PropTypes.string,
+  /**
+   * Passes prop `disabled` to the child `<Textfield>` component
+   */
+  disabled: PropTypes.bool,
+  /**
+   * https://github.com/paypal/downshift#itemtostring
+   *
+   * Used to determine the string value for the selected item (which is used to compute the `inputValue`.
+   */
+  itemToString: PropTypes.func,
+  /**
+   * Array of objects used to populate the suggestion list that appears below the input as users type. This array of objects is intended for an async data callback, and should conform to the prescribed shape to avoid errors.
+   */
+  items: PropTypes.arrayOf(
+    PropTypes.shape({
+      id: PropTypes.string,
+      name: PropTypes.name
+    })
+  ).isRequired,
+  /**
+   * https://github.com/paypal/downshift#onchange
+   *
+   * Called when the user selects an item and the selected item has changed. Called with the item that was selected and the new state of `downshift`.
+   */
+  onChange: PropTypes.func,
+  /**
+   * https://github.com/paypal/downshift#onstatechange
+   *
+   * This function is called anytime the internal state changes. This can be useful if you're using downshift as a "controlled" component, where you manage some or all of the state (e.g. `isOpen, selectedItem, highlightedIndex`, etc) and then pass it as props, rather than letting downshift control all its state itself.
```

Comment:
This documentation is confusing to me, since I'm not sure how one would use Downshift as a "controlled" component through the `<Autocomplete>` component. Do we need this prop?

Reply:
Since we have the TextField as a child now, and can manage our API calls using its `onChange` prop, I think we can remove this.